src/current/_includes/releases/v23.1/v23.1.23.md

```diff
@@ -11,7 +11,7 @@ Release Date: June 20, 2024
     - There were multiple target tables with significant differences in row counts in one changefeed.
     - The changefeed target tables were large with many ranges.
     - The initial scan took a long time to complete (an hour or longer). [#123970][#123970] [#124759][#124759]
-- [Changefeeds]({% link v23.1/change-data-capture-overview.md %}) now default to evenly distributing their work across all [replicas]({% link v23.1/architecture/glossary.md %}#replica), including [followers]({% link v23.1/architecture/replication-layer.md %}#raft), regardless of [leaseholder]({% link v23.1/architecture/glossary.md %}#leaseholder) placement. To disable this behavior, set the [cluster setting]({% link v23.1/cluster-settings.md %}) `changefeed.balanced_distribution.enabled` to `false`. If disabled, changefeed planning reverts to its previous behavior for distributing work. [#124930][#124930]
+- [Changefeeds]({% link v23.1/change-data-capture-overview.md %}) now default to evenly distributing their work across all [replicas]({% link v23.1/architecture/glossary.md %}#replica), including [followers]({% link v23.1/architecture/replication-layer.md %}#raft), regardless of [leaseholder]({% link v23.1/architecture/glossary.md %}#leaseholder) placement. To disable this behavior, set the [cluster setting]({% link v23.1/cluster-settings.md %}) `changefeed.random_replica_selection.enabled` to `false`. If disabled, changefeed planning reverts to its previous behavior for distributing work. [#124930][#124930]
```

Comment:
> now default to

On 23.1 and 23.2, the default is actually `false` for this cluster setting so they'd need to set it to `true` to enable it. The 24.1 documentation looks good!

Reply:
I will wait for @rharding6373's review. The PR description should be updated on the backports too if they list the default incorrectly — for future reference.

Reply:
Andy is correct, this is disabled by default in both 23.1 and 23.2. Thanks for catching this, Andy. I will update the backports and original PR.